### ISSUE TITLE
Release 0.6.8 xpath filter incompatible with previous releases

### ISSUE THREAD

**Issue by calvincheng8** [ncclient/ncclient#416]
The following code used to work in 0.6.7 and earlier,

```
from ncclient import manager

path = '/a/b/c'

with manager.connect(...) as m:
    reply = m.get('xpath', path)
```
Now this code throws an error
```
  File "test-netconf.py", line 7, in <module>
    reply = m.get(filter=('xpath', path))
  File "/usr/local/lib/python3.8/site-packages/ncclient/manager.py", line 226, in execute
    return cls(self._session,
  File "/usr/local/lib/python3.8/site-packages/ncclient/operations/retrieve.py", line 83, in request
    node.append(util.build_filter(filter))
  File "/usr/local/lib/python3.8/site-packages/ncclient/operations/util.py", line 53, in build_filter
    ns, select = criteria
ValueError: too many values to unpack (expected 2)
```

The path parameter in 0.6.7 used to be a string, even the current document still so specifies. But in the unit test and the code the path is expected as a tuple.

Please consider making this backward compatible, and announce such major changes before releasing it.

**Comment by glennmatthews** [comment]
Looks like this change was introduced by #405.

**Comment by fredgan** [comment]
@calvincheng8 Sorry, I will look at it as soon as possible.

**Comment by einarnn** [comment]
I'll disable the 0.6.8 release for now. Apologies for the breaking change. That was unintentional. We have now got admin for ncclient.readthedocs.io, and so we will be trying to get document generation working again.

**Comment by fredgan** [comment]
@calvincheng8 Please try with my PR #418 
It should be fix by that. Sorry for the unexpected incompatible API change again.

**Comment by calvincheng8** [comment]
@fredgan Thanks for the quick fix. Yes the PR #418 works!

**Comment by fredgan** [comment]
Hi @einarnn ,
Resolved by #418, so it should be closed :-)

**Comment by einarnn** [comment]
Resolved by #418
